astro/src/content/json/themes/templates.json

```diff
@@ -1155,7 +1155,7 @@
       {
         "name": "changePasswordId",
         "type": "String",
-        "description": "The id that was sent to the user (usually via email) that allows them to change their password. Normally, this id is included in the email template for the `forgot password` workflow and when the user clicks the link in the email, they are taken to this page with this parameter on the URL."
+        "description": "The Id that was sent to the user (usually via email or SMS) that allows them to change their password. Normally, this Id is included in the template for the `forgot password` workflow and when the user clicks the link in the message, they are taken to this page with this parameter on the URL."
```

Comment:
Noticed this when I was looking for where to link from the message template doc.